pkg/storkctl/clusterpair.go

```diff
@@ -699,14 +711,30 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 	return createClusterPairCommand
 }
 
-func generateClusterPair(name, ns, ip, port, token, configFile, projectIDMappings string, authSecretNamespace string, reverse bool, ignoreStorageOptions bool) (*storkv1.ClusterPair, error) {
+func generateClusterPair(
+	name string,
+	ns string,
+	ip string,
+	port string,
+	token string,
+	configFile string,
+	projectIDMappings string,
+	authSecretNamespace string,
+	reverse bool,
+	mode string,
+	ignoreStorageOptions bool) (*storkv1.ClusterPair, error) {
 	opts := make(map[string]string)
 	if !ignoreStorageOptions {
 		opts["ip"] = ip
 		opts["port"] = port
 		// extract token from px-endpoint command
 		opts["token"] = token
 	}
+	if mode == userInputCPModeAsync {
```

Comment:
ignoreStorageOptions is true in case of sync. But shouldn't we also show mode in the CP in that case just for it to be obvious?

Reply:
We are relying on that information to decide whether the clusterpair is for sync and async cluster. I think it's better to stick with that. 

I agree it's good to have  that value but because `options` key is assumed to be empty for sync cluster, we will break that behavior.